client/extensions/wp-super-cache/contents-tab.js

```diff
@@ -180,19 +188,19 @@ class ContentsTab extends Component {
 
 				<div>
 				{ ! isEmpty( get( wpcache, 'cached_list' ) ) &&
-					<CacheStats header="Fresh WP-Cached Files" files={ wpcache.cached_list } />
+					this.renderCacheStats( wpcache.cached_list, translate( 'Fresh WP-Cached Files' ), true, false )
```

Comment:
Yeah, it seemed like a lot of repetition and I was trying to eliminate some of that. I can change it back though with the suggested change.

Also a question...do you know if there's a way to access `siteId` in `connect` in `ContentsTab` component? It's already been made available by the HOC's `connect`, but isn't available as part of `ownProps` or anything.